web-api/openapi/back_office.yaml

```diff
@@ -44,11 +75,10 @@ paths:
     post:
       tags:
         - back office
-      summary: Add documents to the system.
-      description: |
-        Add documents to the system.
-
-        The system will create a representation of the document that will be used to match it against the preferences of a user.
+        - documents
+      summary: Ingest documents
```

Comment:
i changed all `summary` items to short titles/headlines, as they appear as such in the navigation overview. the `description` items contain full sentences/paragraphs.